node/rpc-api/src/lib.rs

```diff
@@ -54,6 +54,7 @@ pub static ACCESS_MAP: Lazy<HashMap<&str, Access>> = Lazy::new(|| {
     // Wallet API
     access.insert(wallet_api::WALLET_BALANCE, Access::Write);
     access.insert(wallet_api::WALLET_DEFAULT_ADDRESS, Access::Write);
+    access.insert(wallet_api::WALLET_DEFAULT_ADDRESS, Access::Read);
```

Comment:
You probably want to remove one of these lines.

Reply:
Removed